folding-schemes/src/folding/hypernova/cccs.rs

```diff
@@ -92,6 +85,16 @@ impl<F: PrimeField> CCS<F> {
 }
 
 impl<C: CurveGroup> CCCS<C> {
+    pub fn dummy(l: usize) -> CCCS<C>
+    where
+        C::ScalarField: PrimeField,
+    {
+        CCCS::<C> {
+            C: C::zero(),
+            x: vec![C::ScalarField::zero(); l],
+        }
+    }
```

Comment:
I think that for the moment is not needed or at least it would not be used. I'm afraid of falling into *premature abstraction* of adding methods that are not really needed increasing the code complexity without need (not just for this method but in general).